src/__tests__/InputGroup.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import { InputGroup } from '..'
+
+describe('InputGroup', () => {
+  test('renders', () => {
+    const json = renderer.create(<InputGroup />).toJSON()
+    expect(json).toMatchSnapshot()
+  })
```

Comment:
Would you think it'd be a valuable test case to test out the usage of the `borderColor` prop support in an additional unit test code?

Reply:
The `borderColor` function is 100% covered in the styled-system library, so we shouldn't need to test it here IMO